### PR TITLE
Update office-365-encryption-risks-and-protections.md

### PR DESCRIPTION
**Pull request by msbemba**
Modified from 128 to 256 as updated in the document -https://docs.microsoft.com/en-us/microsoft-365/compliance/office-365-bitlocker-and-distributed-key-manager-for-encryption?view=o365-worldwide

Thanks
Sri

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3162

File: microsoft-365/compliance/office-365-encryption-risks-and-protections.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Some risk scenarios and the currently available encryption technologies that mit
...
 | Service Encryption | SharePoint Online | AES 256-bit | The keys used to encrypt the blobs are stored in the SharePoint Online Content Database. The SharePoint Online Content Databases is protected by database access controls and encryption at rest. Encryption is performed using TDE in Azure SQL Database. These secrets are at the service level for SharePoint Online, not at the tenant level. These secrets (sometimes referred to as the master keys) are stored in a separate secure repository called the Key Store. TDE provides security at rest for both the active database and the database backups and transaction logs. When customers provide the optional key, the customer key is stored in Azure Key Vault, and the service uses the key to encrypt a tenant key, which is used to encrypt a site key, which is then used to encrypt the file level keys. Essentially, a new key hierarchy is introduced when the customer provides a key. | Yes |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-encryption-risks-and-protections.md]:
```suggestion
| Service Encryption | SharePoint Online | AES 256-bit | The keys used to encrypt the blobs are stored in the SharePoint Online Content Database. The SharePoint Online Content Database is protected by database access controls and encryption at rest. Encryption is performed using TDE in Azure SQL Database. These secrets are at the service level for SharePoint Online, not at the tenant level. These secrets (sometimes referred to as the master keys) are stored in a separate secure repository called the Key Store. TDE provides security at rest for both the active database and the database backups and transaction logs. When customers provide the optional key, the customer key is stored in Azure Key Vault, and the service uses the key to encrypt a tenant key, which is used to encrypt a site key, which is then used to encrypt the file level keys. Essentially, a new key hierarchy is introduced when the customer provides a key. | Yes |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one additional unrelated edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Comment by Larryngolog** [comment]
Hi! Please also update the "FIPS 140-2 Validated" column for EXO in the table for MultiTenant - since all servers will be using 256 bit key, then I understand that  the annotation "Yes, for servers that use AES 256-bit**" is no longer needed and the column for EXO should just say "Yes" like it does for SPO and S4BO. And this would mean that the footnote "**Most servers in the Exchange Online multi-tenant environment have been deployed with AES 256-bit encryption for BitLocker. Servers using AES 128-bit are being phased out." can also be removed.

**Pull request merged**